apps/telegram/src/main.ts

```diff
@@ -145,7 +146,27 @@ async function subscribeToNotifications() {
   }
 }
 
-function formatMessage({ title, message, url }: PushNotification) {
+async function sendMessageTelegram(
+  chatId: string,
+  notification: PushNotification
+) {
+  const markdown = formatMessageMarkdown(notification);
+  const html = await markdownToTelegramHTMLSafe(markdown);
+  try {
+    telegramBot.sendMessage(chatId, html, {
+      parse_mode: 'HTML',
+      disable_web_page_preview: true,
+    });
+  } catch (error) {
+    console.error(
+      `Error sending message to telegram.\nMarkdown:\n${markdown}\n\nOffending. HTML:\n${html}\n\n`
+    );
```

Comment:
convenient to debug. The method will re-throw, this is why i don't log the error. I just give some context here to debug issues